Document timeout format limitation

When I try to set a timeout to 2month: `P2M` I get an error:
```
[ERROR]         [error]: Build step org.kie.kogito.quarkus.common.deployment.KogitoAssetsProcessor#generateSources threw an exception: org.kie.kogito.codegen.process.ProcessCodegenException: Processes with errors are Invalid process: '/home/kogito/serverless-workflow-project/src/main/resources/m2k.sw.yml'. Found error: java.lang.IllegalArgumentException: An invalid "eventTimeout": "P2M" configuration was provided for the state "WaitForSaveTransformationCompletionEvent" in the serverless workflow: "Move2Kube workflow".When configured, it must be set with a greater than zero ISO 8601 time duration. For example PT30S. Or a valid expression, for example $CONST.myDuration, where 'myDuration' is defined in the constant section of the workflow
```
but according to https://en.wikipedia.org/wiki/ISO_8601#Durations it should be fine

> To resolve ambiguity, "P1M" is a one-month duration and "PT1M" is a one-minute duration (note the time designator, T, that precedes the time value).

According to @wmedvede 
> a P1M (on month) in the context of the workflows can't be translated into how much time to wait.
When we say 1 month, are we talking about January -> 31 days, February -> 29/28 days, etc.

So this needs to be documented

--- a/serverlessworkflow/modules/ROOT/pages/core/timeouts-support.adoc
+++ b/serverlessworkflow/modules/ROOT/pages/core/timeouts-support.adoc
@@ -10,6 +10,11 @@ For example, you can configure how long a workflow can wait for an event to arri
 Regardless of its application scope (workflow or state), the timeouts must be configured as an amount of time (a duration), which is considered to start when the referred scope becomes active. Timeouts use the link:https://en.wikipedia.org/wiki/ISO_8601[`ISO 8601` data and time standard] to specify a duration of time and follow the format `PnDTnHnMn.nS` with days considered to be exactly 24 hours.
 For instance, `PT15M` configures 15 minutes, and `P2DT3H4M` defines 2 days, 3 hours and 4 minutes.
 
+[NOTE]
+====
+Month based timeouts like P2M (period of two months) are not valid since the month duration might vary. In that case you can use PT60D instead.
+====
+
 [#workflow-timeout]
 == Workflow timeout 
 To configure the maximum amount of time a workflow can be running before being canceled, you can use the workflow timeout.

